docs/track.rst

```diff
@@ -682,6 +682,94 @@ Example::
 
 This will shrink the index ``src`` to ``target``. The target index will consist of one shard and have one replica. With ``shrink-node`` we also explicitly specify the name of the node where we want the source index to be relocated to.
 
+delete-ml-datafeed
+~~~~~~~~~~~~~~~~~~
+
+With the operation ``delete-ml-datafeed`` you can execute the `delete datafeeds API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-delete-datafeed.html>`_. The ``delete-ml-datafeed`` operation supports the following parameters:
+
+* ``datafeed-id`` (mandatory): The name of the machine learning datafeed to delete.
+* ``force`` (optional, defaults to ``false``): Whether to force deletion of a datafeed that has already been started.
+
+This runner will intentionally ignore 404s from Elasticsearch so it is safe to execute this runner regardless whether a corresponding machine learning datafeed exists.
+
+This operation works only if `machine-learning <https://www.elastic.co/products/stack/machine-learning>`__ is properly installed and enabled. This is an administrative operation. Metrics are not reported by default. Reporting can be forced by setting ``include-in-reporting`` to ``true``.
+
+create-ml-datafeed
+~~~~~~~~~~~~~~~~~~
+
+With the operation ``create-ml-datafeed`` you can execute the `create datafeeds API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-put-datafeed.html>`__. The ``create-ml-datafeed`` operation supports the following parameters:
+
+* ``datafeed-id`` (mandatory): The name of the machine learning datafeed to create.
+* ``body`` (mandatory): Request body containing the definition of the datafeed. Please see the `create datafeed API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-put-datafeed.html>`__ documentation for more details.
+
+This operation works only if `machine-learning <https://www.elastic.co/products/stack/machine-learning>`__ is properly installed and enabled. This is an administrative operation. Metrics are not reported by default. Reporting can be forced by setting ``include-in-reporting`` to ``true``.
+
+start-ml-datafeed
+~~~~~~~~~~~~~~~~~
+
+With the operation ``start-ml-datafeed`` you can execute the `start datafeeds API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-start-datafeed.html>`__. The ``start-ml-datafeed`` operation supports the following parameters:
+
+* ``datafeed-id`` (mandatory): The name of the machine learning datafeed to start.
+* ``body`` (optional, defaults to empty): Request body with start parameters. Please see the `start datafeed API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-start-datafeed.html>`__ documentation for more details.
+* ``start`` (optional, defaults to empty): Start timestamp of the datafeed. Please see the `start datafeed API <https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-start-datafeed.html>`__ documentation for more details.
```

Comment:
I am not certain if it makes sense to copy/paste the documentation link for every parameter here. Perhaps (for the sake of making the documentation less verbose) we could have the link before the parameters saying ... please see the ... for more details on any of the following parameters.